### PR TITLE
Deploy Translation Service to all non-Prod envs

### PR DESCRIPTION
**Pull request by danlysiak**
### Change description ###
For CHG5008835

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/admin/flux-helm-operator/rbac/aat-role-binding.yaml [hunk 1]
```diff
@@ -619,3 +619,35 @@ subjects:
...
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: flux-helm-operator
+subjects:
+  - name: flux-helm-operator
+    namespace: admin
+    kind: ServiceAccount
+
+---
```
Comment (danlysiak) [inline k8s/namespaces/admin/flux-helm-operator/rbac/aat-role-binding.yaml:637]: Adding for consistency (as should be required for HMC also anyway)

File: k8s/namespaces/admin/flux-helm-operator/rbac/ithc-role-binding.yaml [hunk 2]
```diff
@@ -477,3 +477,35 @@ subjects:
...
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: flux-helm-operator
+subjects:
+  - name: flux-helm-operator
+    namespace: admin
+    kind: ServiceAccount
+
+---
```
Comment (danlysiak) [inline k8s/namespaces/admin/flux-helm-operator/rbac/ithc-role-binding.yaml:495]: Adding for consistency (as should be required for HMC also anyway)

File: k8s/prod/common/ts/identity.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: ts
+  namespace: ts
+spec:
+  type: 0
+  resourceID: "/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourcegroups/managed-identities-aat-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ts-aat-mi"
+  clientID: "30a7b8f4-547a-4c3a-b839-aea6fa8066cf"
```
Comment (danlysiak) [inline k8s/prod/common/ts/identity.yaml:9]: Values to be updated after infra built - dependency on https://github.com/hmcts/cnp-jenkins-config/pull/778

**Review by timja — COMMENTED** [review]
can you do non prod separately to prod please? and verify it's all working first?

**Comment by danlysiak** [comment]
> can you do non prod separately to prod please? and verify it's all working first?

Removed from Prod in https://github.com/hmcts/cnp-flux-config/pull/16990/commits/82db86f2e12291ea35b0c7fc980edd9b338335ca - will re-add later

**Review by timja — APPROVED** [review]

**Review by RebeccaBaker — APPROVED** [review]

**Pull request merged**